src/web/components/ClientSideTokenGeneration/CstgDeleteDomainDialog.stories.tsx

```diff
@@ -8,10 +9,44 @@ const meta: Meta<typeof CstgDeleteDomainDialog> = {
 };
 export default meta;
 
-type Story = StoryObj<typeof CstgDeleteDomainDialog>;
+export const Default = () => {
+  const [isOpen, setIsOpen] = useState<boolean>(false);
+  return (
+    <div>
+      <button className='small-button' type='button' onClick={() => setIsOpen(!isOpen)}>
+        Open Dialog
+      </button>
+      {isOpen && (
+        <CstgDeleteDomainDialog
+          onOpenChange={() => setIsOpen(!isOpen)}
+          domains={['testdomain.com']}
+          onRemoveDomains={() => {
+            Promise.resolve(console.log(`Disabling Domain`));
+            setIsOpen(!isOpen);
```

Comment:
This isn't working.  I think it may need to be before returning the promise like in CstgAddDomainDialog.  It also doesn't work with editing, which follows this pattern.

Reply:
Hmm I'm seeing that these two are working on my end